docs/UPGRADING.md

```diff
@@ -19,6 +19,10 @@ can be freely renamed.
 This change improves the overall code base naming coherence since all the structures involving libraries have the `Name`
 field that refers to the library name as it appears in the `library.properties` file.
 
+#### gRPC message `cc.arduino.cli.commands.v1.Library` no longer has `real_name` field
+
+You must use the `name` field instead.
+
```

Comment:
```suggestion

#### Machine readable `lib list` output no longer has "real name" field

##### JSON

The `[*].library.real_name` field has been removed.

You must use the `[*].library.name` field instead.

##### YAML

The `[*].library.realname` field has been removed.

You must use the `[*].library.name` field instead.

```

Document breaking change to `arduino-cli lib list --format json`, `arduino-cli lib list --format jsonmini`, and `arduino-cli lib list --format yaml` output.